Tests/Marketplace/search_and_install_packs.py

```diff
@@ -365,41 +366,42 @@ def __init__(self, error_msg):
-            response_data, status_code, _ = demisto_client.generic_request_func(client,
-                                                                                path='/contentpacks/marketplace/install',
-                                                                                method='POST',
-                                                                                body={'packs': packs,
-                                                                                      'ignoreWarnings': True},
-                                                                                accept='application/json',
-                                                                                _request_timeout=request_timeout,
-                                                                                response_type='object')
-
-            if status_code in range(200, 300) and status_code != 204:
-                packs_data = [{'ID': pack.get('id'), 'CurrentVersion': pack.get('currentVersion')} for pack in response_data]
-                logging.success(f'Packs were successfully installed on server {host}')
-                logging.debug(f'The packs that were successfully installed on server {host}:\n{packs_data}')
-
-        except ApiException as ex:
+        for attempt in range(attempts_count):
             try:
-                if ex.status in [502, 599]:
-                    if attempts_count <= 1:
-                        raise ex
+                logging.info(f'Installing packs {", ".join([p.get("id") for p in packs_to_install])} on server {host}. '
+                            f'Attempts left on failure: {attempt}.')
+                response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                                    path='/contentpacks/marketplace/install',
+                                                                                    method='POST',
+                                                                                    body={'packs': packs,
+                                                                                        'ignoreWarnings': True},
+                                                                                    accept='application/json',
+                                                                                    _request_timeout=request_timeout,
+                                                                                    response_type='object')
+
+                if status_code in range(200, 300) and status_code != 204:
+                    packs_data = [{'ID': pack.get('id'), 'CurrentVersion': pack.get('currentVersion')} for pack in response_data]
+                    logging.success(f'Packs were successfully installed on server {host}')
+                    logging.debug(f'The packs that were successfully installed on server {host}:\n{packs_data}')
```

Comment:
We want to exit the loop if the installation success.
```suggestion
                    logging.debug(f'The packs that were successfully installed on server {host}:\n{packs_data}')
                    break
```